flink-core/src/main/java/org/apache/flink/configuration/ClusterOptions.java

```diff
@@ -78,4 +79,13 @@
 				.text("Enable the slot spread out allocation strategy. This strategy tries to spread out " +
 					"the slots evenly across all available %s.", code("TaskExecutors"))
 				.build());
+
+	@Documentation.Section(Documentation.Sections.EXPERT_CLUSTER)
+	public static final ConfigOption<Boolean> HALT_ON_FATAL_ERROR =
+		key("cluster.processes.halt-on-fatal-error")
```

Comment:
You will need to regenerate the documentation.

Reply:
I'll update the docs once my computer has finished downloading the internet ;)

Reply:
Currently at:
```
[ERROR] npm ERR! code E404
[ERROR] npm ERR! 404 Not Found: @yarnpkg/lockfile@1.1.0
```